mkdocs.yml

```diff
@@ -95,6 +95,7 @@ nav:
       - 'Metrics Walkthrough': multicluster-gateway-controller/docs/how-to/metrics-walkthrough.md
     - 'Authentication & Authorization':
       - 'AuthPolicy for Application Developers and Platform Engineers': kuadrant-operator/doc/user-guides/auth-for-app-devs-and-platform-engineers.md
+      - 'Authenticated Rate Limiting for Application Developers': kuadrant-operator/doc/user-guides/authenticated-rl-for-app-developers.md
```

Comment:
I think I'd probably add it under the "Rate Limiting" guides instead, like between 'RateLimitPolicy for Platform Engineers' and the more complicated 'authenticated-rl-with-jwt-and-k8s-authnz.md' one. WDYT?

Reply:
Sure ya, that works for me.